frontend/packages/knative-plugin/src/components/services/ServiceRow.tsx

```diff
@@ -58,7 +59,9 @@ const ServiceRow: RowFunction<ServiceKind> = ({ obj, index, key, style }) => {
         {(readyCondition && readyCondition.status) || '-'}
       </TableData>
       <TableData className={tableColumnClasses[7]}>
-        {(readyCondition && readyCondition.message) || '-'}
+        {readyCondition && (
+          <ClampedText lineClamp={5}>{readyCondition?.message || '-'}</ClampedText>
+        )}
```

Comment:
Still need to render `-` if there's no `readyCondition`
```suggestion
        {readyCondition?.message && (
          <ClampedText lineClamp={5}>{readyCondition.message}</ClampedText>
        ) || '-'}
```

Reply:
done